deploy/010-deploy-reward-distributors.ts

```diff
@@ -51,11 +54,15 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
         proxy: {
           implementationName: `RewardsDistributorImpl`,
           owner: proxyOwnerAddress,
-          proxyContract: "OpenZeppelinTransparentProxy",
+          proxyContract: "OptimizedTransparentUpgradeableProxy",
           execute: {
             methodName: "initialize",
             args: [comptrollerProxy.address, rewardTokenAddress, maxLoopsLimit, accessControlAddress],
           },
+          viaAdminContract: {
+            name: "DefaultProxyAdmin",
+            artifact: defaultProxyAdmin,
+          },
           upgradeIndex: 0,
```

Comment:
Don't we need the `maxFeePerGas` property here for zkSync?

Reply:
We need it occasionally. It is due to incorrect fee calculation on zksync sepolia.